CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Upgrade to use semantic conventions 1.6.1 (#7926)
 - `dynatraceexporter`: Validate QueueSettings and perform config validation in Validate() instead (#8020)
 - `signalfxexporter`: Add validation for `sending_queue` setting (#8026)
+- `mongodbatlasreceiver`: modify attributes to adhere to naming guidelines and add 3 new attributes (#7960)
```

Comment:
I think this should be listed as a breaking change

Reply:
You right